src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $main-card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: scale 0.3s;
```

Comment:
The property `scale` is not a valid CSS property for transitions. You likely meant to use `transform` instead.